Cargo.toml

```diff
@@ -17,20 +17,21 @@ bitflags = "1.0"
 cfg-if = "0.1.0"
 void = "1.0.2"
 
-[dependencies.bytes]
-version = "0.4.5"
-# Don't include the optional serde feature
-default-features = false
-
 [target.'cfg(target_os = "dragonfly")'.build-dependencies]
 gcc = "0.3"
 
 [dev-dependencies]
+bytes = { version="0.5.0", default_features = false }
```

Comment:
The `bytes` crate is now only used for examples and tests.  I could've removed it entirely, but since we explicitly supported it in version 0.10.0, I think we ought to use it in the examples as a kind of migration guide.